src/games/behavmixed.imp

```diff
@@ -368,8 +321,6 @@ T MixedBehaviorProfile<T>::GetLiapValue(bool p_definedOnly) const
 
   T x, result = ((T) 0), avg, sum;
 
-  // HACK: force it to recompute data.  FIX THIS.
```

Comment:
Just confirming as an aide-memoire that this comment was referring to the use of this function in `liap_solve`, and that `liap_solve` used to side-step the calls that invalidated these cached data.  So it is completely OK to be rid of this hack now.